Snakefile

```diff
@@ -112,22 +112,24 @@ rule download_seattle_metadata:
 rule download_seattle_sequences:
     message:
         """
-        download_seattle_sequences: Downloading Seattle sequences from fauna
+        download_seattle_sequences: Downloading Seattle sequences from ID3C
         {wildcards.lineage} {wildcards.segment}
         """
     output:
         sequences = "data/seattle_sequences_{lineage}_{segment}.fasta"
     params:
-        fasta_fields = "strain"
+        id3c_url = os.environ['ID3C_SEQ_URL'],
+        id3c_username = os.environ['ID3C_USERNAME'],
+        id3c_password = os.environ['ID3C_PASSWORD']
     shell:
         """
-        python3 {path_to_fauna}/vdb/download.py \
-            --database vdb \
-            --virus seattle \
-            --fasta_fields {params.fasta_fields} \
-            --select segment:{wildcards.segment} type:{wildcards.lineage} \
-            --path data \
-            --fstem seattle_sequences_{wildcards.lineage}_{wildcards.segment}
+        python3 scripts/download_sfs_sequences.py \
+            --id3c-url {params.id3c_url} \
+            --id3c-username {params.id3c_username} \
+            --id3c-password {params.id3c_password} \
```

Comment:
Ditto about the script directly reading the environment variables instead of doing it via params and cli arguments.